src/services/doppler-legacy-client.ts

```diff
@@ -300,7 +302,32 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     });
   }
 
+  private setNoRetryAxios() {
+    axiosRetry(this.axios, { retries: 0 });
+  }
+
+  private setRetryAxiosAndLog() {
+    axiosRetry(this.axios, {
+      retries: 3,
+      shouldResetTimeout: true,
+      retryCondition: () => {
+        return true;
+      },
+      retryDelay: (retryCount: any, error: any) => {
+        addLogEntry({
+          origin: window.location.origin,
+          section: 'Login/legacyClient',
+          browser: window.navigator.userAgent,
+          error: error,
+          retryCount: retryCount,
+        });
+        return retryCount * 1000;
```

Comment:
Do we wait for a second? is it not so much?

Reply:
I don't have a strong opinion on this, maybe less it's ok too. What do you suggest?

Reply:
In my opinion, wait up to 3 seconds is too much.
Maybe waiting only some few milli-seconds?